src/server/live-reload/live-reload-server.js

```diff
@@ -1,26 +1,97 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
+var config = require('../config');
 var fs = require('fs');
+var ncp = require('ncp');
 var path = require('path');
+var prepare = require('../prepare');
+var Q = require('q');
 var telemetry = require('../telemetry-helper');
 var Watcher = require('./watcher').Watcher;
 
+var canRefreshEventName = 'lr-can-refresh';
+var refreshFileEventName = 'lr-refresh-file';
+var fullReloadEventName = 'lr-full-reload';
+
 var socket;
 var watcher;
 
-function cssFileChanged(fileRelativePath) {
-    telemetry.sendTelemetry('live-reload', { fileType: '.css' });
-    socket.emit('lr-update-css', { href: fileRelativePath.replace(/\\/g, '/') });
+function onFileChanged(fileRelativePath, filePathFromProjectRoot) {
```

Comment:
I would break this into multiple functions such that it's a little easier to follow the code.